mmv1/products/bigquery/Dataset.yaml

```diff
@@ -140,6 +140,11 @@ properties:
 
 
             * `allAuthenticatedUsers`: All authenticated BigQuery users.
+        - !ruby/object:Api::Type::String
+          name: 'iamMember'
```

Comment:
We'll need to add a test or modify a test to exercise the new value!

Reply:
@rileykarson I modified the test case with [7352734](https://github.com/GoogleCloudPlatform/magic-modules/pull/9288/commits/73527342031fae8b8d9c214e0260c8e1a2584656)!